### PR TITLE
(Mostly) avoid the term "command" in the docs.

### PR DESCRIPTION
**Pull request by anntzer**
It's a MATLABism; Python has functions and methods.  There are some
places where "command" does read fine, though.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -352,7 +352,7 @@ def legend(self, *args, **kwargs):
 
         Notes
         -----
-        Not all kinds of artist are supported by the legend command. See
+        Some artists are not supported by this function.  See
```
Comment (story645) [inline lib/matplotlib/axes/_axes.py]: I just dig how you cleaned up this construction.

File: lib/mpl_toolkits/axisartist/axislines.py [hunk 2]
```diff
@@ -29,11 +29,11 @@
...
 * label
 
-Note that these are separate artists from Axis class of the
-original mpl, thus most of tick-related command in the original mpl
-won't work, although some effort has made to work with. For example,
-color and markerwidth of the ax.axis["bottom"].major_ticks will follow
-those of Axes.xaxis unless explicitly specified.
+Note that these are separate artists from `matplotlib.axis.Axis`, thus
+most of tick-related functions in Matplotlib won't work, although some
+effort has made to work with. For example, color and markerwidth of the
```
Comment (story645) [inline lib/mpl_toolkits/axisartist/axislines.py]: 'although some effort has made to work with' is very ambigous - what is working with what?

File: tutorials/introductory/pyplot.py [hunk 3]
```diff
@@ -344,10 +339,9 @@ def f(t):
 plt.show()
 
 ###############################################################################
-# All of the :func:`~matplotlib.pyplot.text` commands return an
-# :class:`matplotlib.text.Text` instance.  Just as with with lines
-# above, you can customize the properties by passing keyword arguments
-# into the text functions or using :func:`~matplotlib.pyplot.setp`::
+# All of the `~.pyplot.text` functions return an `matplotlib.text.Text`
```
Comment (story645) [inline tutorials/introductory/pyplot.py]: typo an/a

File: lib/mpl_toolkits/axisartist/axislines.py (hunk 2 above)
Comment (anntzer) [reply lib/mpl_toolkits/axisartist/axislines.py]: let's just remove that phrase

File: tutorials/introductory/pyplot.py (hunk 3 above)
Comment (anntzer) [reply tutorials/introductory/pyplot.py]: fixed

File: lib/mpl_toolkits/axisartist/axislines.py [hunk 4]
```diff
@@ -29,11 +29,10 @@
 * offsetText
 * label
 
-Note that these are separate artists from Axis class of the
-original mpl, thus most of tick-related command in the original mpl
-won't work, although some effort has made to work with. For example,
-color and markerwidth of the ax.axis["bottom"].major_ticks will follow
-those of Axes.xaxis unless explicitly specified.
+Note that these are separate artists from `matplotlib.axis.Axis`, thus most
+of tick-related functions in Matplotlib won't work. For example, color and
```
Comment (story645) [inline lib/mpl_toolkits/axisartist/axislines.py]: doesn't need the of

Comment (anntzer) [reply lib/mpl_toolkits/axisartist/axislines.py]: done

**Pull request merged**